awscli/data/cli.json

```diff
@@ -52,8 +52,12 @@
             "help": "<p>Do not sign requests.  Credentials will not be loaded if this argument is provided.</p>"
         },
         "ca-bundle": {
-                "dest": "ca_bundle",
-                "help": "<p>The CA certificate bundle to use when verifying SSL certificates. Overrides config/env settings.</p>"
+            "dest": "ca_bundle",
+            "help": "<p>The CA certificate bundle to use when verifying SSL certificates. Overrides config/env settings.</p>"
+        },
+        "cli-read-timeout": {
```

Comment:
Oh.  I'm guessing that because the args are defined in json, you can't set `type: int` like you normally could with `add_argument`?